b2sdk/bucket.py

```diff
@@ -493,6 +493,7 @@ def upload_bytes(
         encryption: Optional[EncryptionSetting] = None,
         file_retention: Optional[FileRetentionSetting] = None,
         legal_hold: Optional[LegalHold] = None,
+        cache_control: Optional[str] = None,
```

Comment:
if someone called it like this:

```
bucket.upload_bytes(
        self,
        data_bytes,
        file_name,
        content_type,
        file_infos,
        progress_listener,
        encryption,
        file_retention,
        legal_hold,
        large_file_sha1,
)
```
And you add `cache_control` as non-last argument, then the caller will suddenly find his positional argument to mis-align and fail due to a sha1 being accidentally put into the new `cache_control` variable instead of the old `large_file_sha1`. After your change every user of the sdk would have to adjust their code to make sure this doesn't happen. We would need to release a major version, 2.0.0.

Instead, please put the new argument at the very end of the argument list.

(if someone is using named arguments, that would still work, but we can't guarantee that every usr of the library does that)